llvm/lib/Target/X86/X86InstrSSE.td

```diff
@@ -6699,28 +6699,30 @@ let Constraints = "$src1 = $dst" in {
                                    null_frag>, REX_W;
   }
 
+  let Predicates = [In64BitMode] in {
   def CRC32r32m8_EVEX  : SS42I_crc32m<0xF0, "crc32{b}", GR32, i8mem,
-                                      int_x86_sse42_crc32_32_8>, EVEX, T_MAP4, PS;
+                                      int_x86_sse42_crc32_32_8>, EVEX, NoCD8, T_MAP4, PS;
```

Comment:
Remove PS, it's redundant.

Reply:
It's needed, base class CRC32I has defined XD.